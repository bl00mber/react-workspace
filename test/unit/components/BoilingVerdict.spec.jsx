import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import BoilingVerdict from '@/components/BoilingVerdict';

describe('BoilingVerdict', () => {
  test('should match enzyme shallow snapshot', () => {
    const wrapper = shallow(<BoilingVerdict celsius={0} />);
    expect(toJson(wrapper)).toMatchSnapshot();
    wrapper.setProps({ celsius: 100 });
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
