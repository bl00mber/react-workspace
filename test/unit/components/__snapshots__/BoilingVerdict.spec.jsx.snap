// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BoilingVerdict should match enzyme shallow snapshot 1`] = `
<p>
  The water would not boil.
</p>
`;

exports[`BoilingVerdict should match enzyme shallow snapshot 2`] = `
<p>
  The water would boil.
</p>
`;
